### PR TITLE
Tweaks to program and sponsors page

### PR DESCRIPTION
**Pull request by jsheunis**

### CODE REVIEW

File: _data/program_2021.yml [hunk 1]
```diff
@@ -130,11 +139,11 @@
...
         startTime: "13:00",
         endTime: "13:45",
-        sessionTitle: "TBD",
-        speaker: "TBD",
+        sessionTitle: "Data visualization demo",
+        speaker: "To be announced",
         speakerLink: "../page-speakers",
         breakStatus,
-        sessionType: "TBD",
+        sessionType: "Workshop",
```
Comment (DorienHuijser) [inline _data/program_2021.yml]:
```suggestion
        sessionType: "Demo",
```

**Review by DorienHuijser — CHANGES_REQUESTED** [review]
All looks good, minor suggestion!

**Review by jsheunis — COMMENTED** [review]
great, thanks!

**Comment by jsheunis** [comment]
all good!

**Pull request merged**